### ISSUE TITLE
Child permissions no longer being applied (?)

### ISSUE THREAD

**Issue by andrewkm** [Arnuh/EchoPet#146]
Cannot seem to figure this one out.
Child permissions no longer seem to get applied.
I've tested both our fork as well as the latest official development build here:
https://jenkins.arnah.ca/job/EchoPet-Dev/119/

```
  echopet.pet.baby.*:
    default: op
    description: 'All baby permissions'
    children:
      echopet.pet.type.axolotl.baby: true
      echopet.pet.type.bee.baby: true
      echopet.pet.type.camel.baby: true
      echopet.pet.type.cat.baby: true
      echopet.pet.type.chicken.baby: true
      echopet.pet.type.cow.baby: true
      echopet.pet.type.donkey.baby: true
      echopet.pet.type.drowned.baby: true
      echopet.pet.type.fox.baby: true
      echopet.pet.type.goat.baby: true
      echopet.pet.type.hoglin.baby: true
      echopet.pet.type.horse.baby: true
      echopet.pet.type.husk.baby: true
      echopet.pet.type.llama.baby: true
      echopet.pet.type.mule.baby: true
      echopet.pet.type.mushroomcow.baby: true
      echopet.pet.type.ocelot.baby: true
      echopet.pet.type.panda.baby: true
      echopet.pet.type.pig.baby: true
      echopet.pet.type.piglin.baby: true
      echopet.pet.type.pigzombie.baby: true
      echopet.pet.type.polarbear.baby: true
      echopet.pet.type.rabbit.baby: true
      echopet.pet.type.sheep.baby: true
      echopet.pet.type.skeletonhorse.baby: true
      echopet.pet.type.sniffer.baby: true
      echopet.pet.type.strider.baby: true
      echopet.pet.type.traderllama.baby: true
      echopet.pet.type.turtle.baby: true
      echopet.pet.type.villager.baby: true
      echopet.pet.type.wolf.baby: true
      echopet.pet.type.zoglin.baby: true
      echopet.pet.type.zombie.baby: true
      echopet.pet.type.zombiehorse.baby: true
      echopet.pet.type.zombievillager.baby: true
```

This as well is true for `hats.*`, etc.
TL;DR applying `echopet.pet.baby.*` no longer gives the child permissions above.
I've no idea when this broke, but we've begun getting reports post our 1.20 update (we went from 1.19.4 -> 1.20.1) and sadly I have no way to test on earlier versions to perhaps find the breaking commit.

I've verified that other plugins have their child permissions applying correctly to ensure there's no issue with my permissions environment, just for my sanity to be perfectly sure. (Tested with mcmmo child permissions, work just fine.)

Setup:
EchoPet: https://jenkins.arnah.ca/job/EchoPet-Dev/119/
Paper 1.20.1: Build 161
LuckPerms: 5.4.102

Let me know if you need anymore information.
Been trying to figure this out for a good 1-2 hours now, and just about gave up lol.

**EDIT:** _Just to rule out conflicting plugins, I've also tried running JUST the above three, paper, echopet and luckperms, nothing more. Same issue._

**EDIT-2:** _Tried several previous luckperms versions to rule that out as well, including as far back as a year ago, issue persists._

**Comment by Arnuh** [comment]
Probably due to this https://github.com/Arnuh/EchoPet/commit/507c3cd3181624af88f59502e18b67594f03789f

Just realized they don't actually document permissions at all for this so maybe try copy pasting them from plugin.yml? or just deleting paper-plugin from the jar
